test/e2e/dataplane/tcp_gn_pod_connectivity.go

```diff
@@ -85,4 +85,28 @@ var _ = Describe("[dataplane-globalnet] Basic TCP connectivity tests across over
 			verifyInteraction(framework.GatewayNode, framework.NonGatewayNode)
 		})
 	})
+
+	When("a pod connects via TCP to the globalIP of a remote headless service", func() {
+		BeforeEach(func() {
+			toEndpointType = tcp.GlobalPodIP
+			networking = framework.PodNetworking
+		})
+
+		When("the pod is not on a gateway and the remote service is not on a gateway", func() {
+			verifyInteraction(framework.NonGatewayNode, framework.NonGatewayNode)
+		})
+
+		When("the pod is not on a gateway and the remote service is on a gateway", func() {
+			verifyInteraction(framework.NonGatewayNode, framework.GatewayNode)
+		})
+
+		When("the pod is on a gateway and the remote service is not on a gateway", func() {
+			verifyInteraction(framework.GatewayNode, framework.NonGatewayNode)
+		})
```

Comment:
Since 
```suggestion
```

Since we are adding many new tests with GN2.0, we can avoid certain tests to limit the e2e test duration, particularly the ones we believe will be covered by other tests. So, lets remove these two tests.